exercises/practice/parallel-letter-frequency/src/main/java/ParallelLetterFrequency.java

```diff
@@ -2,12 +2,12 @@
 
 class ParallelLetterFrequency {
 
-    ParallelLetterFrequency(String letters) {
+    ParallelLetterFrequency(String[] letters) {
         throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
     }
 
-    Map<Integer, Integer> letterCounts() {
+    Map<Character, Integer> letterCounts() {
```

Comment:
Maybe rename this method to `countLetters()` or `getLetterCounts()` since we are changing it anyway.

Reply:
Tell me if this is still required after reverting again to ``<Integer, Integer>``